tgui/packages/tgui/interfaces/CharacterPreferences/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { useBackend, useLocalState, useSharedState } from '../../backend';
+import {
+  Box,
+  Button,
+  ByondUi,
+  Divider,
+  Flex,
+  Icon,
+  LabeledList,
+  NoticeBox,
+  Section,
+  Stack,
+  Tabs,
+} from '../../components';
+import { Window } from '../../layouts';
+import { CharacterPreferencesData, CharacterPreferencesProfile, CharacterPreferencesTabKeys } from './type';
+import { SavesTab } from './SavesTab';
+import { CharacterTab } from './CharacterTab';
+import { GeneralTab } from './GeneralTab';
+import { GameSettingsTab } from './GameSettingsTab';
+import { Fragment } from 'inferno';
+
+export const CharacterPreferences = (_props, context) => {
```

Comment:
Why props are underscored?

Reply:
Convention used by TypeScript if you need access to a later argument and don't make use of the earlier ones.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals